packages/auto-approve/src/get-pr-info.ts

```diff
@@ -130,3 +133,35 @@ export async function getReviewsCompleted(
   });
   return reviewsCompleted.data as Reviews[];
 }
+
+/**
+ * Function grabs contents of a specific file in a repository
+ * @param owner of repo of pr
+ * @param repo of of repo of pr
+ * @param path path to file
+ * @returns content of the file, or else throws an error if not a file
+ */
+export async function getFileContent(
+  owner: string,
+  repo: string,
+  path: string,
+  octokit: Octokit
+): Promise<string> {
+  const fileContent = await octokit.repos.getContent({
+    owner,
+    repo,
+    path,
+  });
+
+  if (isFile(fileContent.data)) {
+    const dataContent = Buffer.from(
+      fileContent.data.content,
+      'base64'
+    ).toString('utf8');
+    return dataContent;
+  }
+
+  throw new Error(
+    `${owner}/${repo}/${path} is a folder, cannot get file contents`
+  );
```

Comment:
nit(personal preference): prefer throwing the error in the guard statement and having the success case at the bottom.